### PR TITLE
eslint cleanup

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

**Comment by phillimorland** [comment]
@khamui failing tests?

**Comment by khamui** [comment]
not sure why it is not failing locally. Is there other eslint configurations on the servers?

File: package.json [hunk 1]
```diff
@@ -29,10 +29,8 @@
     "eslint-config-standard-jsx": "11.0.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-n": "15.2.3",
-    "eslint-plugin-node": "11.1.0",
```
Comment (phillimorland) [inline package.json:32]: @khamui do we want to enable these others too or no?

**Review by phillimorland — COMMENTED** [review]
looking good and linting fine, just questions about what appear to be unused imports

File: package.json (hunk 1 above)
Comment (khamui) [reply package.json:32]: its the same as `eslint-plugin-n`, that was forked because `...-node` is not maintained anymore

File: .eslintrc [hunk 2]
```diff
@@ -1,28 +1,55 @@
 {
+  "parser": "@babel/eslint-parser",
```
Comment (khamui) [inline .eslintrc:2]: eslintrc is now identical to aplus

File: package.json (hunk 1 above)
Comment (khamui) [reply package.json:32]: ah, yes added back and activated with the latest commit :)

File: .eslintrc [hunk 3]
```diff
@@ -1,25 +1,55 @@
...
   "rules": {
-    "jsx-quotes": [2, "prefer-double"]
+    "jsx-quotes": [2, "prefer-double"],
+    "jsx-a11y/no-onchange": "off",
+    "react/prop-types": "off",
+    "n/no-missing-require": "off",
+    "n/no-unsupported-features/es-syntax": "off",
+    "n/no-missing-import": "off",
+    "n/no-unpublished-import": "off",
+    "n/no-extraneous-import": "off"
```
Comment (phillimorland) [inline .eslintrc:34]: is it recommended to switch off these rules or is it just because our code doesn't work when they are on? :p

**Review by phillimorland — APPROVED** [review]
very nice! :)

File: .eslintrc (hunk 3 above)
Comment (phillimorland) [reply .eslintrc:34]: ok saw the answer in a+:)

**Review by goapunk — APPROVED** [review]
very nice!

**Pull request merged**